docs/wiki/Whats-new.md

```diff
@@ -2,7 +2,6 @@
 
 - [In this Section](#in-this-section)
 - [Updates](#updates)
-  - [November 2022](#november-2022)
   - [October 2022](#october-2022)
```

Comment:
Missing the Nov 2022 TOC item @JamJarchitect

Reply:
🤦‍♂️ Added it now. Thanks @jtracey93